pcsx2/CDVD/CDVDaccess.cpp

```diff
@@ -368,12 +368,20 @@ bool DoCDVDopen()
 
 	auto CurrentSourceType = enum_cast(m_CurrentSourceType);
 	int ret = CDVD->open(!m_SourceFilename[CurrentSourceType].IsEmpty() ?
-							 static_cast<const char*>(m_SourceFilename[CurrentSourceType].ToUTF8()) :
-							 (char*)NULL);
+                             static_cast<const char*>(m_SourceFilename[CurrentSourceType].ToUTF8()) :
+                             (char*)NULL,
```

Comment:
Tabs instead of spaces, above too.

Reply:
Apparently, clang-format prefers spaces. fixed...

Reply:
Need to adjust that clang setting then.